jekyll/_cci2_ja/creating-orbs.md

```diff
@@ -18,119 +18,66 @@ version:
-
-1. **プルリクエストのタイトルに特別なセマンティック バージョン タグを付けます。**<br/>含まれている CI 設定では、[semver ]({{site.baseurl}}/2.0/orb-concepts/#semantic-versioning)リリースで指定された正しいタグがコミットメッセージに含まれている場合、[ orb-tools Orb ](https://circleci.com/developer/orbs)を使用して、デフォルトブランチでテストに合格した Orb を自動的にパブリッシュします。<br/>タグテンプレートは、`[semver:<increment>]` のようになります。`<increment>` は、次のいずれかの値に置き換えられます。
+**注:**プライベート Orb の場合、Orb レジストリでは検索できません。 しかし、その Orb の認証ユーザーは URL にアクセスすることができます。
+{: class="alert alert-warning"}
 
-    | increment の値 | 説明                             |
-    | ------------ | ------------------------------ |
-    | major        | リリースのバージョン番号を 1.0.0 だけ増やして公開する |
-    | minor        | リリースのバージョン番号を x.1.0 だけ増やして公開する |
-    | patch        | リリースのバージョン番号を x.x.1 だけ増やして公開する |
-    | skip         | リリースを公開しない                     |
-    {: class="table table-striped"}
-
-    たとえば、`alpha` ブランチから Orb のメジャーバージョン リリースを初めて公開する場合は、プルリクエストのタイトルを `[semver:major] 初回 Orb リリース` のように設定します。 ![Orb の初回メジャー リリース - プル リクエスト]({{site.baseurl}}/assets/img/docs/orb_semver_release_pr.png)
-
-1. **「スカッシュ」マージ**<br/>[スカッシュ](https://docs.github.com/ja/github/collaborating-with-issues-and-pull-requests/about-pull-request-merges#squash-and-merge-your-pull-request-commits) マージを行うと、デフォルトのブランチへのマージ時にブランチが 1 つのコミットにまとめられるだけでなく、プルリクエストのタイトルがコミットメッセージとして維持されます。
+![Orb のパブリッシュプロセス]({{ site.baseurl }}/assets/img/docs/orb-publishing-process.png)
 
-    マージする前に`[semver:<increment>]` が以下のように、コミットメッセージのサブジェクト/先頭行にあることを確認してください。 Bitbucket では、プルリクエストのタイトルがコミットメッセージのサブジェクト/先頭行ではなく本文に残っている場合があります。 ![プルリクエストをスカッシュマージして semver タイトルを保持する]({{site.baseurl}}/assets/img/docs/orb_semver_squash_merge.png)
+[手動]({{site.baseurl}}/ja/2.0/orb-author-validate-publish)ではなく、[Orb 開発キット]({{site.baseurl}}/ja/2.0/orb-author/#orb-development-kit)を使用して Orb をパブリッシュすると、セマンティックリリースがこのセクションで説明する手順で簡単に行えます。 パブリッシュプロセスの概説は、オーサリングプロセスの開始時に `circleci orb init` コマンドで生成される [README.md](https://github.com/CircleCI-Public/Orb-Template/blob/main/README.md) ファイルを参照してください。
 
-1. **完了です！**<br/>[CircleCI アプリケーション](https://app.circleci.com/)にアクセスすると、Orb のパブリッシュ パイプラインの進捗状況を確認できます。 このパイプラインが完了したら、[Orb レジストリ](https://circleci.com/developer/ja/orbs) に Orb が公開されます。
+## Orb 開発キットを使った新リリースの公開
+{: #issue-a-new-release-with-the-orb-development-kit }
 
-### Orb のパブリッシュプロセス
-{: #orb-publishing-process }
+以下では、Orb の新しいセマンティックリリースを公開する方法について説明します。
 
-ここでは、Orb 開発キットについて掘り下げ、Orb のパブリッシュに関係するコンポーネントについて説明します。
+`circleci orb init` コマンドでサンプルの Orb プロジェクトを生成すると、自動的に `alpha` ブランチに移行されます。 ブランチ名は任意で構いませんが、デフォルト以外のブランチで変更することをお勧めします。
 
-[circleci orb init]({{site.baseurl}}/2.0/orb-author/#getting-started) コマンドは、最適な Orb 開発パイプラインで設計された定義済みの CircleCI 設定ファイルを含む [Orb テンプレートリポジトリ](https://github.com/CircleCI-Public/Orb-Project-Template)を、Orb 用にクローンします。
+変更したら、コミットしブランチにプッシュします。 コミットメッセージには、[Conventional Commits](https://www.conventionalcommits.org/) を使用することを強くお勧めします。
```

Comment:
📝 https://www.conventionalcommits.org/ja
  It looks like they have Japanese page as well. Might be better to reference for usability.

Reply:
Thank you! I've added /ja.